.github/workflows/ci.yml

```diff
@@ -336,7 +312,6 @@ jobs :
       - ktlint
       - lint
       - publish-maven-local
-      - publish-snapshot
```

Comment:
Is there another way to include the snapshot in the all-checks job?

I'm wondering if this change is worth it since we can already publish manual snapshots when needed prior to the change hitting main

Reply:
We can't do that anymore, with the recent changes to Square's OSS publishing.